### PR TITLE
Modifying example to search command subfolders

### PR DESCRIPTION
**Pull request by JonathanRJoyner**
This portion of the guide is confusing because it does not work with a previous section title ["Command Categories"](https://discordjs.guide/creating-your-bot/command-handling.html#command-categories) which has the user move commands in to subcommand folders.

The changes I've made allow for the full commands folder to be searched for a matching command. This will work no matter the path to the file inside the command folder.

### CODE REVIEW

**Comment by GodderE2D** [comment]
This would require users to download the glob package using npm. As this is not stated on the page, I would recommend adding a part for installing the package.

**Comment by JonathanRJoyner** [comment]
Good point. Added a line about installing glob.

File: guide/additional-features/reloading-commands.md [hunk 1]
```diff
@@ -14,8 +14,13 @@ This section assumes you followed the [Command Handling](/creating-your-bot/comm
 The reload command ideally should not be used by every user. You should deploy it as a guild command in a private guild.
 :::
 
+We will use the glob package to navigate our slash command file structure. This can be installed via npm by running `npm i glob` in the terminal.
```
Comment (megatank58) [inline guide/additional-features/reloading-commands.md:17]:
```suggestion
You can use the glob package to navigate your slash command file structure. To install the package run `npm install glob` in the terminal.
```
wording and consistency https://github.com/discordjs/guide/blob/main/CONTRIBUTING.md#youyour-instead-of-weour

**Comment by GodderE2D** [comment]
Wait, you're making this change to reflect the command categories page, right? So why should we use a different approach than that page? I'd say either use `fs` here, or also make the categories page use `glob`.

**Comment by sdanialraza** [comment]
I don't think changes like these are needed especially when the new guide is in the works.

**Comment by Jiralite** [comment]
> This portion of the guide is confusing because it does not work with a previous section title ["Command Categories"](https://discordjs.guide/creating-your-bot/command-handling.html#command-categories) which has the user move commands in to subcommand folders.

Just pointing out that this issue is resolved.

**Comment by Jiralite** [comment]
Node.js is adding this [natively](https://nodejs.org/en/blog/announcements/v22-release-announce#glob-and-globsync), so it'd probably be a good idea to use this when it's stable. For now, keeping things as-is is likely best. Thank you for your contribution, nonetheless.

**Pull request closed without merging**